docker-compose.prod.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+
+
+services:
+
+
+    db_drugs:
+        image: mysql:8
+        container_name: db_drugs
+        environment:
+            MYSQL_DATABASE: drugs
+            MYSQL_USER: admin
+            MYSQL_PASSWORD: admin
+            MYSQL_ROOT_PASSWORD: supersecret
+        volumes:
+            - db_open_data:/var/lib/mysql
+            - ./data:/data
+        command:
+            - '--secure-file-priv=/data'
+        
+    serve_drugs:
+        container_name: serve_drugs
+        environment:
+          DB_URI: 'mysql+pymysql://admin:admin@db:3306/drugs'
+        build:
+            context: .
+            dockerfile: Dockerfile.serve
+        depends_on:
+            - db_drugs
+        links:
+            - "db_drugs:db"
```

Comment:
I believe links are actually legacy now in docker, we should consider removing this in the future. Not a necessary change in this PR but something to consider moving forward.